### DISCUSSION TITLE
Test Case.

### DISCUSSION THREAD

**Discussion by 0ME9A** [codinasion-archive/codinasion-monorepo#4540] [Ideas]
Hello everyone,

Welcome to our new codination/program discussions.

## Solution: I have an idea that can help us improve the quality of our programs and eliminate bugs. We should incorporate multiple test cases into our testing process. I encourage you all to raise any questions or provide suggestions on how we can achieve a bug-free program.

**Comment by harshraj8843** [comment]
Hii 😍 

Please let us know...

**Comment by harshraj8843** [comment]
> our program only has one test case

I'm not sure that we have any test cases for now  ¯\\\_(ツ)\_/¯

**Reply by 0ME9A** [reply]
🤔

**Reply by 0ME9A** [reply]
what is this:-  ¯\_(ツ)_/¯

**Reply by 0ME9A** [reply]
Issue no #2810 soluteion #4538 coded for this:-
```
 Input  : 12:05:45PM
 Output : 12:05:45

```
Fail in:-
```
Input : 5:5:5pm
Output : 725:5

```

**Reply by harshraj8843** [reply]
Updating all existing issues with multiple test cases is not an optimal solution in my opinion

We can

### 1. Update all existing issues

- Update all existing issues (using GitHub API)
- Consider adding more test cases in new issues

### 2. Experiment

I always thought of creating a separate test case for testing all the codes.

But due to having multiple languages, I can't think of an optimal solution for this.

**Reply by 0ME9A** [reply]
Creating issues for update test-case🤔


### Update all existing issues 👍✔️
* Update all existing issues (using GitHub API)
* Consider adding more test cases in new issues

**Reply by 0ME9A** [reply]
What do you think about creating new issues for updating test cases? Do you think it would be helpful in keeping track of changes and improving the testing process? 🤖🤖🤖

**Comment by 0ME9A** [comment]
Are you (@harshraj8843) alive? 🤔

**Reply by harshraj8843** [reply]
Kind of 😆
